app/bundles/CoreBundle/Assets/css/libraries/bootstrap/less/alerts.less

```diff
@@ -18,6 +18,7 @@
     // Specified for the h4 to prevent conflicts of changing @headings-color
     color: inherit;
   }
+
```

Comment:
Why are file permissions changed here and on several other files?

Reply:
Good catch.
Fixed bd75f3e54cf064a16857ed6a62c15637632db648